Libraries/test/Amazon.Lambda.Annotations.SourceGenerators.Tests/Snapshots/Greeter_SayHelloAsync_Generated.g.cs

```diff
@@ -19,9 +19,9 @@ public Greeter_SayHelloAsync_Generated()
         public async Task<APIGatewayProxyResponse> SayHelloAsync(APIGatewayProxyRequest request, ILambdaContext context)
         {
             var firstNames = default(System.Collections.Generic.IEnumerable<string>);
-            if (request.Headers?.ContainsKey("names") == true)
+            if (request.MultiValueHeaders?.ContainsKey("names") == true)
             {
-                firstNames = request.Headers["names"].Split(",").Select(q => (string)Convert.ChangeType(q, typeof(string))).ToList();
+                firstNames = request.MultiValueHeaders["names"].Select(q => (string)Convert.ChangeType(q, typeof(string))).ToList();
```

Comment:
Snapshot was wrong because of a bug in creating FromHeader attribute.

Arguments are passed in constructor than property initializer.